.changeset/poor-planes-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
+---
+
+Remove function call from logged_in_user object in template
```

Comment:
Removed from where? What template?

(I know the answers to these from this PR, but it will be hard to tell when this is part of [our changelog](https://v-next--cloudfour-patterns.netlify.app/?path=/story/changelog--page)

Reply:
```suggestion
Remove function call from logged_in_user object properties in Comment Form template
```